.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
This file is from `mvn -N io.takari:maven:wrapper` https://github.com/takari/maven-wrapper.

Reply:
I'm surprised this isn't available as a jar on maven central.